WcaOnRails/app/views/incidents/index.html.erb

```diff
@@ -0,0 +1,108 @@
+<% provide(:title, "Incidents log") %>
+
+<%= render layout: "nav" do %>
+  <h1><%= yield(:title) %></h1>
+
+  <div id="custom-search-toolbar">
+    <input id="incident-tags" class="form-control" type="text" placeholder="Filter by tags">
+  </div>
+  <div class="incidents-log-table-container">
+    <%= wca_table(table_class: "incidents-log-table", data: { toggle: "table", pagination: true, search: true, toolbar: "#custom-search-toolbar" }, floatThead: false, greedy: false) do %>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th class="tags-cell">Tags</th>
+          <th class="comps-cell">Happened during</th>
+          <th class="status-cell">Status</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @incidents.each do |incident| %>
+          <tr>
+            <td><%= link_to incident.name, incident %></td>
+            <td data-tags="<%= incident.tags %>">
+              <% incident.tags_array.each do |t| %>
+                <%= render "incident_tag", tag: t %>
+              <% end %>
+            </td>
+            <td data-tags="<%= incident.competitions %>">
+              <% incident.competitions.each do |c| %>
+                <%= render "competition_tag", comp: c %>
+              <% end %>
+            </td>
+            <td><%= incident.status %></td>
+          </tr>
+        <% end %>
+      </tbody>
+    <% end %>
+  </div>
+
+  <script>
+var resetSelectedTags = function(e, tag) {
+  e.preventDefault();
+  $('.search input').val("");
+  var selectize = $('input#incident-tags')[0].selectize;
+  selectize.clear();
+  selectize.addItem(tag);
+}
+
+
+var bootstrap_object = null;
```

Comment:
Everything below here is getting defined in the global scope, which has potential to mess stuff up. I'd feel safer if this was wrapped in an [IIFE](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression)?